contracts/FutarchyTemplate.sol

```diff
@@ -1,13 +1,129 @@
 pragma solidity 0.4.24;
 
+import "@aragon/templates-shared/contracts/TokenCache.sol";
 import "@aragon/templates-shared/contracts/BaseTemplate.sol";
 
-contract FutarchyTemplate is BaseTemplate {
-  constructor(DAOFactory _daoFactory, ENS _ens, MiniMeTokenFactory _miniMeFactory, IFIFSResolvingRegistrar _aragonID)
-  BaseTemplate(_daoFactory, _ens, _miniMeFactory, _aragonID)
+contract FutarchyTemplate is BaseTemplate, TokenCache {
+  string constant private ERROR_EMPTY_HOLDERS = "COMPANY_EMPTY_HOLDERS";
+  string constant private ERROR_BAD_HOLDERS_STAKES_LEN = "COMPANY_BAD_HOLDERS_STAKES_LEN";
```

Comment:
company bad holders stakes len??

Reply:
lol

Reply:
@emaG3m this is all copied from https://github.com/aragon/dao-templates/blob/master/templates/company/contracts/CompanyTemplate.sol